### ISSUE TITLE
selector-attribute-brackets-space-inside

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1086]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Options: "always" | "never"

Both valid:

```css
a[ href^="http" ] {
    color: #ff0000;
}
```

```css
a[href^="http"] {
    color: #ff0000;
}
```

**Comment by jeddy3** [comment]
I couldn't quickly find the acceptance of whitespace in the specs, but it does work in Chrome.

I'm happy to see this included, unless there are any objections?

- Name: `selector-attribute-square-brackets-space-inside`.
- Primary options: `"always"|"never"`.
- Secondary options: none.

**Comment by davidtheclark** [comment]
LGTM. Should we just call them `brackets` instead of `square-brackets`? `selector-attribute-brackets-space-inside`?

**Comment by evilebottnawi** [comment]
@davidtheclark yep

**Comment by jeddy3** [comment]
Agreed.

- Name: `selector-attribute-brackets-space-inside`.
- Primary options: `"always"|"never"`.
- Secondary options: none.

**Comment by evilebottnawi** [comment]
Merged. Close.
